### ISSUE TITLE
pb文件

### ISSUE THREAD

**Issue by SentimentSongs** [MaybeShewill-CV/lanenet-lane-detection#177]
pb文件是不是还得 通过
tensorflow-r1.9/tensorflow/python/tools/freeze_graph.py
脚本转换

bazel build tensorflow/python/tools:freeze_graph && \
bazel-bin/tensorflow/python/tools/freeze_graph \
--input_graph=some_graph_def.pb \
--input_checkpoint=model.ckpt-8361242 \
--output_graph=/tmp/frozen_graph.pb --output_node_names=softmax

里边有input_graph 这个pb 是不是放训练生成的pb

**Comment by MaybeShewill-CV** [comment]
@SentimentSongs 需要你自己frezze model或者你可以使用tensorflow现在的标准格式 saved model, 具体可以去tensorflow查api:)
